sdk/src/tools/code-search.ts

```diff
@@ -1,11 +1,11 @@
 import { spawn } from 'child_process'
 import * as path from 'path'
 
-import { rgPath } from '@vscode/ripgrep'
+import { getRgPath } from '@codebuff/common/native/ripgrep'
```

Comment:
We've been importing all common files with relative imports FYI. This only works if we publish @codebuff/common as another npm package